articles/ai-services/cognitive-services-support-options.md

```diff
@@ -25,6 +25,11 @@ Explore the range of [Azure support options and choose the plan](https://azure.m
 * [Azure portal](https://portal.azure.com/#blade/Microsoft_Azure_Support/HelpAndSupportBlade/overview)
 * [Azure portal for the United States government](https://portal.azure.us)
 
+>[!NOTE]
+>
+> You can click your Azure AI resource in *Help + Support* in Azure Portal, *Support + Troubleshooting* under *Help*, then input your brief question in the text box under the question "Tell us about the issue to get solutions and support." You can find *Cognitive Services* and your resource are correctly selected.
```

Comment:
I don't actually see these options. What Azure portal page are users supposed to be on at this step?

Reply:
I replied about it in the chat between you and me. Can you check it?